src/utils/envSpec.ts

```diff
@@ -1,4 +1,4 @@
-import envUtils from './env';
+import * as envUtils from './env';
 
 export default describe('env utils', () => {
     const VALID_PC_TLDS = [
```

Comment:
I could go either way here.  Keeping this list up-to-date or just having tests for the unique domain patterns we have here so we get out of the business of tracking this list.  Thoughts?

Reply:
Ah yeah, didn't realize we had tests around the tlds. I think I'm in favor of killing these and not tracking the list.